mdn-short-surveys/2024-08-05-web-apps/README.md

```diff
@@ -0,0 +1,98 @@
+# Web App Short Survey
+
+## Date
+Survey run between 2024-08-05 - 2024-08-23 to 5% of MDN users.
+
+## Questions
+
+1. In the past year, have you built an installable web application? _(single selection)_
+  * Yes
+  * No
+  * Don't know
+
+2. In your experience, are web applications sufficiently capable to replace native apps on these platforms? _(each option listed with 6 choices: "Don't know", "Not at all capable", "Not very capable", "Somewhat capable", "Very capable", "Completely capable")_
+  * Android
+  * iOS
+  * macOS
+  * Windows
+
+3. In your experience, what is the biggest challenge building web applications? _(free-form text)_
+
+## Results
+
+### Response Counts
+
+*   Completion Rate: 99.9%
+*   Complete: 752
+*   Partial: 1
+*   Totals: 753
+
+### Q1 - In the past year, have you built an installable web application?
+
+| Value       | Percent | Responses |
+| ----------- | ------- | --------- |
+| Yes         | 34.2%   | 257       |
+| No          | 56.1%   | 421       |
+| Don't know  | 9.7%    | 73        |
+| Totals      |         | 751       |
+
+
+### Q2 - In your experience, are web applications sufficiently capable to replace native apps on these platforms?
+
+| Platform | Don't know | Not at all capable | Not very capable | Somewhat capable | Very capable | Completely capable | Responses |
+| -------- | ---------- | ------------------ | ---------------- | ---------------- | ------------ | ------------------ | --------- |
+| Android  | 46 (19.0%) | 15 (6.2%)          | 15 (6.2%)        | 69 (28.5%)       | 73 (30.2%)   | 24 (9.9%)          | 242       |
+| iOS      | 50 (21.4%) | 32 (13.7%)         | 36 (15.4%)       | 68 (29.1%)       | 36 (15.4%)   | 12 (5.1%)          | 234       |
+| macOS    | 62 (26.7%) | 22 (9.5%)          | 23 (9.9%)        | 56 (24.1%)       | 45 (19.4%)   | 24 (10.3%)         | 232       |
+| Windows  | 39 (16.0%) | 17 (7.0%)          | 13 (5.3%)        | 58 (23.8%)       | 76 (31.1%)   | 41 (16.8%)         | 244       |
+| Totals   |            |                    |                  |                  |              |                    | 244       |
+
+### Q3 - In your experience, what is the biggest challenge building web applications? (free-form)
+The following challenges were extracted from the free-form responses:
```

Comment:
It should be possible to link to the other file:
```suggestion
The following challenges were extracted from the [free-form responses](free-form-responses.md):
```

Reply:
Thanks Francois! I've asked about the csv, and will share once I get it.